release_notes/topics/4_0_0_final.adoc

```diff
@@ -1,3 +1,15 @@
-= Test Feature
+= Client Scopes and suport for OAuth 2 scope parameter
 
-Some test feature.
\ No newline at end of file
+We added support for Client Scopes. Client Scopes replaced Client Templates. They allow more flexibility and better support for `scope` parameter,
+which we never properly supported until now! In relation to this, there are also some changes on consent screen. The items on consent
+screen are now links to client scopes instead of protocol mappers and roles as it was before. See our documentation and migration guide
+for more details.
+
+= OAuth 2 Certificate Bound Access Tokens
+
+Thanks to community contribution from Takashi Norimatsu, we know partially implement specification
```

Comment:
Start with "We know have a partial implementation of the specification .."

Reply:
We now have...

Reply:
"We now have..."